web/core/components/issues/relations/issue-list.tsx

```diff
@@ -0,0 +1,53 @@
+"use client";
+import React, { FC, Fragment } from "react";
+import { observer } from "mobx-react";
+import { TIssue, TIssueRelationTypes } from "@plane/types";
+// components
+import { RelationIssueListItem } from "@/components/issues/relations";
+// types
+import { TRelationIssueOperations } from "../issue-detail-widgets/relations/helper";
+
+type Props = {
+  workspaceSlug: string;
+  projectId: string;
+  issueId: string;
+  issueIds: string[];
+  relationKey: TIssueRelationTypes;
+  issueOperations: TRelationIssueOperations;
+  handleIssueCrudState: (key: "update" | "delete", issueId: string, issue?: TIssue | null) => void;
+  disabled?: boolean;
+};
+
+export const RelationIssueList: FC<Props> = observer((props) => {
+  const {
+    workspaceSlug,
+    projectId,
+    issueId,
+    issueIds,
+    relationKey,
+    disabled = false,
+    issueOperations,
+    handleIssueCrudState,
+  } = props;
+
+  return (
+    <div className="relative">
+      {issueIds &&
+        issueIds.length > 0 &&
+        issueIds.map((relationIssueId) => (
+          <Fragment key={relationIssueId}>
```

Comment:
why fragment here? you can pass the key directly?

Reply:
Resolved